### PR TITLE
Add documentation for log injection

### PR DESCRIPTION
**Pull request by flands**
Note: this is dependent on https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/1200 shipping and the distribution being updated

### CODE REVIEW

**Comment by iNikem** [comment]
@flands Do you plan to update this PR?

**Comment by mateuszrzeszutek** [comment]
FYI: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/logger-mdc-instrumentation.md

File: README.md [hunk 1]
```diff
@@ -114,7 +114,12 @@ System property values take priority over corresponding environment variables.
 Documentation on how to manually instrument a Java application are available
 [here](https://github.com/open-telemetry/opentelemetry-java-instrumentation#manually-instrumenting).
 
-## Troubleshooting the Java Agent
+## Connecting traces and logs
```
Comment (brianashby-sfx) [inline README.md:117]:
```suggestion
## Inject trace IDs and span IDs in logs
```

Comment (flands) [inline README.md:117]:
```suggestion
## Inject trace and span ID into logs
```

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by iNikem** [comment]
I would merge it when we sync this repo to instrumentation 0.9.0 when it is released

**Pull request merged**